ctakesclient/text2fhir.py

```diff
@@ -472,3 +472,20 @@ def nlp_fhir(
         as_fhir.append(nlp_procedure(subject_id, encounter_id, docref_id, match, source))
 
     return as_fhir
+
+def text2fhir(physician_note: str, polarity: Polarity = Polarity.pos) -> List[DomainResource]:
```

Comment:
`text2fhir.text2fhir()` is a little redundant, but no worse than `ctakesclient.client.extract` I suppose